src/collection/store/actions.ts

```diff
@@ -59,4 +93,28 @@ const setCollectionLoading = (id: string): SetCollectionLoadingAction => ({
 	loading: true,
 });
 
-export { setCollectionSuccess, setCollectionError, setCollectionLoading, getCollection };
+const setCollectionsSuccess = (data: Avo.Collection.Response[]): SetCollectionsSuccessAction => ({
+	data,
+	type: CollectionsActionTypes.SET_COLLECTIONS_SUCCESS,
+});
+
+const setCollectionsError = (): SetCollectionsErrorAction => ({
+	type: CollectionsActionTypes.SET_COLLECTIONS_ERROR,
```

Comment:
Moet `loading: true` hier niet nog bij? Anders blijft die open staan na het initializen van die `setCollectionsLoading`, toch?

Reply:
`loading` wordt in de reducer juist gezet.